tests/dataset_readers/transformer_squad_test.py

```diff
@@ -0,0 +1,69 @@
+import pytest
```

Comment:
No test for the model?

Reply:
Added in 2e9bb9cd2c4c55c56fb8942aeb15068fcde07558 and friends.